testpython.py

```diff
@@ -0,0 +1,97 @@
+import os
+import json
+
+def validate_json_structure(data):
```

Comment:
This is only valid for the `JSON_ADE/boards/board_name/board_name.json` but you can get the measurement paths and the packets path in it and then use the actual logic in those files, check [the JSON_ADE repo](https://github.com/HyperloopUPV-H8/JSON_ADE) for the structure